statsmodels/tsa/tests/test_stattools.py

```diff
@@ -879,13 +879,55 @@ def test_innovations_algo_filter_kalman_filter(reset_randomstate):
     res = mod.filter(np.r_[ar_params, ma_params, sigma2])
 
     # Test that the two approaches are identical
+    assert_allclose(u, res.forecasts_error[0])
+    assert_allclose(theta[1:, 0], res.filter_results.kalman_gain[0, 0, :-1])
+    assert_allclose(llf_obs, res.llf_obs)
```

Comment:
im confused as to where these lines came from.  are these related to zivot_andrews?

Reply:
Hmmmm. This one looks related to the mangled merge. Must have missed these three lines during the rebase. Will fix this first.

Reply:
Fixed this issue along with the simple 'filename' change below. Will wait to make sure there are no further merge problems before starting on the rest of these.